guestbook-archetype/src/main/resources/archetype-resources/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# How to become a contributor and submit your own code
```

Comment:
I included this file because I originally intended to use this archetype to build one of our repo (appengine-guestbook-java). However, on the second thought, since this will potentially become user's project, we should remove this file.